kge/job/entity_ranking.py

```diff
@@ -44,6 +47,17 @@ def _prepare(self):
             pin_memory=self.config.get("eval.pin_memory"),
         )
 
+        # assign a tie breaker
+        tie_handling = self.config.get("entity_ranking.tie_handling")
+        if tie_handling == "rounded_mean_rank":
```

Comment:
And move this to get_ranks. Gives us more flexibility later on, if we want to use better strategies (such as mean_metric).